skbio/stats/ordination/_principal_coordinate_analysis.py

```diff
@@ -196,6 +222,14 @@ def pcoa(
         sum_eigenvalues = np.sum(eigvals)
 
     proportion_explained = eigvals / sum_eigenvalues
+    if 0 < number_of_dimensions < 1:
+        cumulative_variance = np.cumsum(proportion_explained)
+        num_dimensions = (
+            np.searchsorted(cumulative_variance, number_of_dimensions, side="left") + 1
```

Comment:
```suggestion
            np.searchsorted(cumulative_variance, number_of_dimensions, side="right")
```

?

Reply:
We tested this and found that left + 1 produces the correct result at edge cases (e.g., the cumulative variance equal to the parameter).